challenge-2/submissions/evassilyev/solution-template.go

```
@@ -0,0 +1,31 @@
+		// Call the ReverseString function
+		output := ReverseString(input)
+
+		// Print the result
+		fmt.Println(output)
+	}
+}
+
+// ReverseString returns the reversed string of s.
+func ReverseString(s string) string {
+	// TODO: Implement the function
+    result := make([]rune, len(s), len(s))
```

golangci-lint (2.13.2)

[error] S1019: should use make([]rune, len(s)) instead

(staticcheck)